Documentation/ceph-csi-snapshot.md

```diff
@@ -99,3 +101,82 @@ kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/pvc-restore.yaml
 kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/snapshot.yaml
 kubectl delete -f cluster/examples/kubernetes/ceph/csi/rbd/snapshotclass.yaml
 ```
+
+## CephFS Snapshots
+
+**NOTE: At present, there is a limit of 400 snapshots per cephFS filesystem. Also PVC cannot be deleted if it's having snapshots.
+make sure all the snapshots on the PVC are deleted before you delete the PVC.**
+
+### SnapshotClass
+
+In [snapshotClass](https://github.com/rook/rook/tree/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/cephfs/snapshotclass.yaml),
+the `csi.storage.k8s.io/snapshotter-secret-name` parameter should reference the
+name of the secret created for the cephfsplugin.
+
+In the snapshotclass, update the value of the `clusterID` field to match the namespace that Rook is
```

Comment:
When mentioning the properties, how about adding it as a bullet list and start each bullet with the setting name? In the paragraph form it is easy to overlook the setting names. This suggestion applies to all the sections where you have new settings to document. I guess this also affects the RBD docs, so this could be a separate PR.

Reply:
Sure.. we could do a follow-up PR for this.